service/src/main/java/org/mskcc/cmo/metadb/service/impl/SampleServiceImpl.java

```diff
@@ -157,7 +156,7 @@ public PublishedMetadbSample getPublishedMetadbSample(UUID metadbSampleId) throw
     public List<PublishedMetadbSample> getPublishedMetadbSampleListByCmoPatientId(
             String cmoPatientId) throws Exception {
         List<SampleMetadata> sampleMetadataList = sampleRepository
-                .findAllSampleMetadataListByCmoPatientId(cmoPatientId);
+                .findAllResearchSampleMetadataByCmoPatientId(cmoPatientId);
         List<PublishedMetadbSample> samples = new ArrayList<>();
         for (SampleMetadata sample: sampleMetadataList) {
             // TODO: update method to fill in sample details in a more inclusive
```

Comment:
This will be resolved when we have ResearchSampleService and ClinicalSampleService